File: 17s2/test09/uniq_echo.py
#!/usr/bin/python3

import sys

word_set = []
data_set = sys.argv[1:]
for item in data_set:
	if item not in word_set:
		word_set.append(item)
    
for item in word_set:
	print(item, end = " ")
print()

File: 17s2/test07/uniq_echo.pl
#!/usr/bin/perl -w

#print ("@ARGV\n");
#print ("$ARGV[0]\n");
#print ("$ARGV[1]\n");
#print ("$ARGV[2]\n");
@tem = ();
foreach $item (@ARGV) {
	if (grep {$_ eq $item} @tem) {
		next;
	}
	push @tem, $item;
}

print "@tem\n";

	

